test/infrastructure/docker/internal/provisioning/ignition/kindadapter.go

```diff
@@ -32,6 +32,18 @@ import (
 	"sigs.k8s.io/cluster-api/test/infrastructure/docker/internal/provisioning"
 )
 
+const (
+	kubeproxyComponentConfig = `
+---
+apiVersion: kubeproxy.config.k8s.io/v1alpha1
+kind: KubeProxyConfiguration
+conntrack:
+# Skip setting sysctl value "net.netfilter.nf_conntrack_max"
+# It is a global variable that affects other namespaces
+	maxPerCore: 0
```

Comment:
```suggestion
  maxPerCore: 0
```

Looks like we use two spaces instead of tab in the other one. No idea if relevant